src/components/CategorySelector.tsx

```diff
@@ -93,19 +96,36 @@ export const CategorySelector = ({
           InputProps={{ endAdornment: <SearchIcon /> }}
           onChange={(event) => setSearchValue(event.target.value)}
         />
-        <Box
-          sx={{
-            display: 'flex',
-            gap: '0.5rem',
-            alignItems: 'center',
-          }}>
-          <FilterVintageIcon
+        {enableNviHighlightning ? (
+          <Chip
+            icon={
+              <FilterVintageIcon
+                color="primary"
+                titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
+                fontSize="small"
+              />
+            }
             color="primary"
```

Comment:
Chip burde kanskje ha en `title` eller `<Tooltip>` som sier hva som skjer når man trykker på? Ala noe sånt:
```suggestion
            color="primary"
            title="Velg alle NVI-kategorier"
```

Reply:
Mulig det holder med "Velg alle" også, siden det står "Kan vi publiseringspoeng" i chipen 🤔